config/config.sample.php

```diff
@@ -2118,6 +2118,17 @@
 	 */
 	'mysql.collation' => null,
 
+	/**
+	 * PostgreSQL SSL connection
+	 */
+	'pgsql.ssl' => [
```

Comment:
```suggestion
	'pgsql_ssl' => [
```

https://docs.nextcloud.com/server/latest/developer_manual/basics/storage/configuration.html#naming-conventions